dist/win/contrib/patchHosts.bat

```diff
@@ -0,0 +1,3 @@
+@echo off
+cd %~dp0
+powershell -NoLogo -NonInteractive -ExecutionPolicy Unrestricted -Command .\patchHosts.ps1
```

Comment:
I guess we can change `-ExecutionPolicy Unrestricted`, as the script should get signed in official builds. Not in local builds, though.

Reply:
To test it with a CI build, we need to change it.

Reply:
why do we **need** to change it? The current version will work regardless of where it is built. The question is, do we want a **stricter** policy, once the ps1 script gets signed?

I would keep it as is for now but make some experiments with the signed script in a later version.

Reply:
Ok, i thought that maybe we can already make the point here and not delay this decision. When there is not a ticket, it has a high probability that we'll never do change/test it.

Reply:
I'll create an issue